Sources/Client/Model/Channel+Requests.swift

```diff
@@ -0,0 +1,457 @@
+    /// - Parameters:
+    ///   - message: a message.
+    ///   - completion: a completion block with `FlagMessageResponse`.
+    @discardableResult
+    func flag(message: Message, _ completion: @escaping Client.Completion<FlagMessageResponse>) -> URLSessionTask {
+        guard config.flagsEnabled else {
+            completion(.success(.init(messageId: message.id, created: Date(), updated: Date())))
+            return .empty
+        }
+        
+        return message.flag(completion)
+    }
+    
+    /// Unflag a message.
+    /// - Parameters:
+    ///   - message: a message.
+    ///   - completion: a completion block with `FlagMessageResponse`.
+    @discardableResult
+    func unflag(message: Message, _ completion: @escaping Client.Completion<FlagMessageResponse>) -> URLSessionTask {
+        guard config.flagsEnabled else {
+            completion(.success(.init(messageId: message.id, created: Date(), updated: Date())))
+            return .empty
+        }
+        
+        return message.unflag(completion)
+    }
+    
+    // MARK: - Members
+    
+    /// Add a member to the channel.
+    /// - Parameters:
+    ///   - member: a member.
+    ///   - completion: a completion block with `ChannelResponse`.
+    @discardableResult
+    func add(_ member: Member, _ completion: @escaping Client.Completion<ChannelResponse>) -> URLSessionTask {
+        return add(Set([member]), completion)
+    }
+    
+    /// Add members to the channel.
+    /// - Parameters:
+    ///   - members: members.
+    ///   - completion: a completion block with `ChannelResponse`.
+    @discardableResult
+    func add(_ members: Set<Member>, _ completion: @escaping Client.Completion<ChannelResponse>) -> URLSessionTask {
+        var members = members
+        
+        self.members.forEach { existsMember in
+            if let index = members.firstIndex(of: existsMember) {
+                members.remove(at: index)
+            }
+        }
+        
+        if members.isEmpty {
+            completion(.success(.init(channel: self)))
+            return .empty
+        }
+        
+        return Client.shared.request(endpoint: .addMembers(members, self), completion)
+    }
+    
+    /// Remove a member from the channel.
+    /// - Parameters:
+    ///   - member: a member.
+    ///   - completion: a completion block with `ChannelResponse`.
+    @discardableResult
+    func remove(_ member: Member, _ completion: @escaping Client.Completion<ChannelResponse>) -> URLSessionTask {
+        return remove(Set([member]), completion)
+    }
+    
+    /// Remove members from the channel.
+    /// - Parameters:
+    ///   - members: members.
+    ///   - completion: a completion block with `ChannelResponse`.
+    @discardableResult
+    func remove(_ members: Set<Member>, _ completion: @escaping Client.Completion<ChannelResponse>) -> URLSessionTask {
+        var existsMembers = Set<Member>()
+        
+        self.members.forEach { existsMember in
```

Comment:
Instead of this, I suggest:
`let existingMembers = members.intersection(self.members)`

https://developer.apple.com/documentation/swift/set/1779150-intersection